.github/workflows/cmake/dependencies_mac.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 Axel Huebl
+#
+# License: BSD-3-Clause-LBNL
+
+# search recursive inside a folder if a file contains tabs
+#
+# @result 0 if no files are found, else 1
+#
+
+set -eu -o pipefail
+
+brew update
+brew install openmpi
```

Comment:
Let's try for now to install dependencies with homebrew.
if this should be too unstable, we will find another solution. but I am using it in CI elsewhere and it's actually quite good.